scripts/cxg_remaster.py

```diff
@@ -0,0 +1,155 @@
+import argparse
+import time
+
+import numpy as np
+import psutil
+import tiledb
+
+"""
+Re-master an existing "X" tiledb array in a CXG
+Creates a new array with the same data but a different tdb schema
```

Comment:
`different` how?

Reply:
based on the parameters set by the user. will add a comment